scripts/lib/version-discovery.mjs

```
@@ -0,0 +1,96 @@
+import { existsSync, readdirSync, readFileSync } from "node:fs";
+import { join } from "node:path";
+
+/**
+ * Plugin-folder suffixes whose `manifest.json` carries the Elgato-style
+ * `Version` field the release bumps. Discovery is anchored to these suffixes
+ * rather than "any `packages/*​/<dir>/manifest.json` that declares a string
```

ESLint

[error] Irregular whitespace not allowed.

(no-irregular-whitespace)